pages/department/[name].tsx

```diff
@@ -0,0 +1,75 @@
+import { text2color } from 'idea-react';
+import { InferGetServerSidePropsType } from 'next';
+import { compose, errorLogger, translator } from 'next-ssr-middleware';
+import { PureComponent } from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+
+import { CommentBox } from '../../components/Base/CommentBox';
+import { LarkImage } from '../../components/Base/LarkImage';
+import { GroupCard } from '../../components/Department/Card';
+import PageHead from '../../components/Layout/PageHead';
+import { i18n } from '../../models/Base/Translation';
+import { Personnel, PersonnelModel } from '../../models/Personnel';
+import { Department, DepartmentModel } from '../../models/Personnel/Department';
+
+export const getServerSideProps = compose<
+  { name: string },
+  { department: Department; personnels: Personnel[] }
+>(errorLogger, translator(i18n), async ({ params: { name } = {} }) => {
+  const [department] = await new DepartmentModel().getList({ name });
+
+  if (!department) return { notFound: true, props: {} };
+
+  const personnels = await new PersonnelModel().getList({
+    department: name,
+  });
+
+  return {
+    props: JSON.parse(JSON.stringify({ department, personnels })),
+  };
+});
+
+export default class DepartmentDetailPage extends PureComponent<
```

Comment:
```suggestion
import { observer } from 'mobx-react';

@observer
export default class DepartmentDetailPage extends PureComponent<
```
国际化文本的更新需要组件重绘，因此需要变为**数据观察者**。